### PR TITLE
Screenshot utilities added to TestUtils

### PR DESCRIPTION
**Pull request by ke-de-ve**

### CODE REVIEW

File: src/test/java/EntityFieldsTest.java [hunk 1]
```diff
@@ -5,15 +5,17 @@
...
 import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Ignore;
 import org.testng.annotations.Test;
 import runner.BaseTest;
 
 import java.util.UUID;
 
 public class EntityFieldsTest extends BaseTest {
 
+    @Ignore
```
Comment (Stan575) [inline src/test/java/EntityFieldsTest.java:16]: This test fixed in main, src/test/java/EntityFieldsTest.java don't need any changes

File: src/test/java/ScreenshotTest.java [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+        browser.get("https://www.wikipedia.org");
+
+        WebElement searchLang = browser.findElement(By.xpath("//select[@name='language']/option[@value='ru']"));
+        searchLang.click();
+
+        WebElement input = browser.findElement(By.xpath("//input[@id='searchInput']"));
+        input.sendKeys("Обеспечение качества");
+
+        // imitate successful assert (happy path)
+        Assert.assertNotEquals(input.getText(), "ZZZZZZZZZ");
```
Comment (Stan575) [inline src/test/java/ScreenshotTest.java:29]: for happy assertion you can use Assert.assertTrue(true)

File: src/test/java/ScreenshotTest.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+
+        // just take screenshot after successful assert (happy path)
+        TestUtils.makeScreenShot(browser, "/tmp/dmitryTestRun_01.png");
+
+        // imitate failed assert to test makeScreenShotAfterTest method
+        //Assert.assertEquals(input.getText(), "Обеспечение качества");
```
Comment (Stan575) [inline src/test/java/ScreenshotTest.java:35]: Avoid commented out code in repository

File: src/test/java/ScreenshotTest.java [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+/**
+ * example of how to use TestUtils.makeScreenShot
+ */
+public class ScreenshotTest extends BaseTest {
+
+    /***
+     * demonstration of how to use makeScreenShot within the test
+     */
+    @Test
+    public void takeScreenshotDemo() {
```
Comment (Stan575) [inline src/test/java/ScreenshotTest.java:17]: I ran this demo test locally, no png files found after, probably there is a problem with path destination location

**Review by Stan575 — CHANGES_REQUESTED** [review]

File: src/test/java/ScreenshotTest.java (hunk 4 above)
Comment (OxanaE) [reply src/test/java/ScreenshotTest.java:17]: if you run it on your local create tmp directory in the root. Mac/Linux:   /tmp  windows: c:\tmp

File: src/test/java/ScreenshotTest.java (hunk 2 above)
Comment (ke-de-ve) [reply src/test/java/ScreenshotTest.java:29]: merged with current main

File: src/test/java/ScreenshotTest.java (hunk 3 above)
Comment (ke-de-ve) [reply src/test/java/ScreenshotTest.java:35]: removed comment

File: src/test/java/ScreenshotTest.java (hunk 4 above)
Comment (ke-de-ve) [reply src/test/java/ScreenshotTest.java:17]: locally you need to create directory where it stores file.  if you have Mac/Linux  then create /tmp .  if you  have windows and running from drive C then create c:\tmp.

        TestUtils.makeScreenShot(browser, "/tmp/dmitryTestRun_01.png");

to test picture on failure at the end add assert that will fail :
Assert.assertTrue(false);

File: src/test/java/EntityFieldsTest.java (hunk 1 above)
Comment (ke-de-ve) [reply src/test/java/EntityFieldsTest.java:16]: merged with current main, so this no longer will be part of request.

File: src/test/java/ScreenshotTest.java (hunk 2 above)
Comment (ke-de-ve) [reply src/test/java/ScreenshotTest.java:29]: :-) thanks. changed to use that

File: src/test/java/ScreenshotTest.java (hunk 3 above)
Comment (ke-de-ve) [reply src/test/java/ScreenshotTest.java:35]: removed comment and replaced with explanation

**Pull request closed without merging**